karma.conf.js

```
@@ -32,23 +35,24 @@ module.exports = function (config) {
     preprocessors: {
       'test/**/*.js': ['browserify']
     },
-    browserify: {
+    browserify: Object.assign({
+      insertGlobalVars: bundlerOptions.insertGlobalVars
+    }, {
       debug: true,
       configure: function configure (b) {
-        b.ignore('glob')
-          .ignore('fs')
-          .ignore('path')
-          .ignore('supports-color')
-          .require(path.join(__dirname, 'node_modules', 'buffer'), {expose: 'buffer'})
+        build(b)
           .on('bundled', function (err, content) {
-            if (!err && bundleDirpath) {
+            if (err) {
+              throw err;
```

apparently we were eating errors before... oops.